infinity/formats/itm_v1.py

```diff
@@ -485,7 +628,7 @@ class ITM_V10_Format (Format):
             { 'key': 'opcode_number',
               'type': 'WORD',
               'off': 0x0000,
-              'enum': 'effects',
+              'enum': 'effects', # FIXME: this ids file does not exist
               'label': 'Opcode number'},
```

Comment:
they just need to be found, but I doubt iesh has any extra search paths one could set. The ones for ees will still need to be created, but the bg2 one can be used as a start. GemRB also has its own overrides, but that can be ignored here.